pkg/kubelet/prober/worker.go

```diff
@@ -98,16 +101,27 @@ func newWorker(
 		w.initialValue = results.Success
 	}
 
-	w.proberResultsMetricLabels = prometheus.Labels{
-		"probe_type":     w.probeType.String(),
-		"container_name": w.container.Name,
```

Comment:
Woah, this is weird. I realize you didn't write this, but curious if anyone knows why we would have multiple labels on a metric which would always share the same label value?

Reply:
Are you talking about container and contaiber_name?

Reply:
Either. The previous implementation of this seems to store the same value for in both the container label and the container_name label. Same thing happens for pod and pod_name, which I find quite odd.

Reply:
You can see it when you examine the actual metric output via curl. 

```# TYPE prober_probe_result gauge
prober_probe_result{container="etcd-container",container_name="etcd-container",namespace="kube-system",pod="etcd-server-events-kubernetes-master",pod_name="etcd-server-events-kubernetes-master",pod_uid="1234",probe_type="Liveness"} 0```

Why do we have two labels which always have the same value in either of them?

Reply:
Super weird.

Reply:
This was for migration purposes. For a long time these and cadvisor metrics used pod_name and container_name label keys, which violate the instrumentation guidelines. For migration purposes in 1.14 both are present.

Reply:
Ah, that would explain it.